kivymd/uix/button/button.kv

```diff
@@ -134,12 +134,12 @@
         adaptive_size: True
         padding: 0
         spacing: "4dp"
+        orientation: 'vertical' if root.icon_position == 'top' else 'horizontal'
 
         MDIcon:
             id: lbl_ic
             size_hint_x: None
-            pos_hint: {"center_y": .5}
-            icon: root.icon
+            pos_hint: {"center_x": .5} if root.icon_position == 'top' else {'center_y': .5}            icon: root.icon
```

Comment:
Here I see a syntax error.

Reply:
Sorry for missing, I'm surprised I didn't get any errors from running it. Will fix shortly.